patterns/pagination.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Title: Default Pagination 
+ * Slug: course/pagination
+ * Inserter: yes
```

Comment:
I don't think we should make this available in the inserter, as it only makes sense to use pagination inside the Query block.

Reply:
Fixed here 768c45c283a7102034ce7172d08e9aa442e0eb2e